source/import-pipeline-from-text.txt

```diff
@@ -24,18 +27,39 @@ pipeline stages using the Pipeline Builder's :guilabel:`Output` panes.
 Syntax
 ------
 
-The imported pipeline must be in the
-:manual:`MongoDB query language </tutorial/query-documents/>`
-(i.e. the same syntax used as the ``pipeline`` parameter of the
-:manual:`db.collection.aggregate()
-</reference/method/db.collection.aggregate/>` method). The imported
-pipeline must be an array, even if there is only one stage in the
-pipeline.
+The imported pipeline must be:
+
+- In the :manual:`MongoDB query language </tutorial/query-documents/>`
+  (i.e. the same syntax used as the ``pipeline`` parameter of the
+  :manual:`db.collection.aggregate()
+  </reference/method/db.collection.aggregate/>` method).
+- Must be an array, even if there is only one stage in the pipeline.
```

Comment:
We already have "must be" in the lead-in to this bulleted list, so I think we can remove it here.